build_tools/pkgci/external_test_suite/pytorch_models_cpu_llvm_task.json

```diff
@@ -14,7 +14,9 @@
   "expected_compile_failures": [
     // TODO(#17344): need to regenerate .mlirbc
     "opt-125M",
-    "resnet50",
+    "resnet50"
   ],
-  "expected_run_failures": []
+  "expected_run_failures": [
+    "sdxl-vae-decode-tank"
+  ]
```

Comment:
Aw, not great that this regressed on CPU

Reply:
yeah vae-decode and scheduled-unet on cpu regressed :(